tests/unittests/cleaner_tests.py

```diff
@@ -274,3 +279,35 @@ def test_ad_username(self):
         line = "DOMAIN\myusername"
         _test = self.uname_parser.parse_line(line)[0]
         self.assertNotEqual(line, _test)
+
+
+class PrepperTests(unittest.TestCase):
+    """
+    Ensure that the translations for different parser/mapping methods are
+    working
+    """
+
+    def setUp(self):
+        self.prepper = SoSPrepper()
+        self.archive = SoSReportArchive(
+            archive_path='tests/test_data/sosreport-cleanertest-2021-08-03-qpkxdid.tar.xz',
+            tmpdir='/tmp'
+        )
+        self.host_prepper = HostnamePrepper()
+        self.ipv4_prepper = IPPrepper()
+
+    def test_parser_method_translation(self):
+        self.assertEqual([], self.prepper.get_parser_file_list('hostname', None))
```

Comment:
Isn't it simpler to merge `test_ipv4_prepper_parser_files` into here? I.e. to have one method with a few asserts "a parser X has file_list Y"? And _maybe_ one assert per each one parser (if our focus here is to have coverage of parsers rather than coverage of `get_parser_file_list` translation "only")?